cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -351,28 +354,31 @@ func mountPartitionMatchingKernelDisk(dir, fallbacklabel string) error {
 	return doSystemdMount(partSrc, dir, opts)
 }
 
-func generateMountsCommonInstallRecover(mst *initramfsMountsState, recoverySystem string) error {
+func generateMountsCommonInstallRecover(mst *initramfsMountsState) error {
 	// 1. always ensure seed partition is mounted first before the others,
 	//      since the seed partition is needed to mount the snap files there
 	if err := mountPartitionMatchingKernelDisk(boot.InitramfsUbuntuSeedDir, "ubuntu-seed"); err != nil {
 		return err
 	}
 
+	// load model and verified essential snaps metadata
```

Comment:
s/verified/verify/

Reply:
verified was intentional,  we are loading and verifying so "load verified..."